src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/ConfigReader.java

```diff
@@ -0,0 +1,22 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.deciders.Decider;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class ConfigReader {
+    public static HashMap<Decider, ArrayList<String>> getDeciderActionPriorityOrder() {
+        ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
```

Comment:
Is there any specific reason to from from YML instead of JSON ? All RCA related configs are stored in JSON file and why do we want to switch to a different format in decision maker ?